lib/lexisnexis/response.rb

```diff
@@ -49,9 +49,13 @@ def handle_unexpected_verification_status_error
     def handle_verification_transaction_error
       return unless verification_status == 'error'
 
+      conversation_id = response_body.dig('Status', 'ConversationId')
+      reference = response_body.dig('Status', 'Reference')
       error_code = response_body.dig('Status', 'TransactionReasonCode', 'Code')
       error_information = response_body.fetch('Information', {}).to_json
-      message = "Response error with code '#{error_code}': #{error_information}"
+
+      message = "(LN ConversationId: #{conversation_id}; Reference: #{reference}) " +
```

Comment:
Any reason not to separate out the tracking ids in the response, as you do in the parser?

Reply:
Nope. Just did that second and forgot to clean it up behind.